crates/tick/src/fmt/mod.rs

```diff
@@ -15,6 +15,12 @@
 //! - [`UnixSeconds`]: Parsing and formatting of system time that is represented as the number of whole seconds since Unix epoch.
 //!   For example, `0` represents `Thu, 1 Jan 1970 00:00:00 -0000`.
 //!
+//! # Extension Traits
+//!
+//! - [`SystemTimeExt`]: Extension trait for [`SystemTime`][`std::time::SystemTime`] that provides convenient
```

Comment:
Shouldn't we generalize this a bit? That is, instead of .display() maybe it should be display_as_8601 and then we can have similar methods for all the formats?